src/main/java/com/microsoft/lst_bench/input/BenchmarkObjectFactory.java

```diff
@@ -77,6 +77,7 @@ private static JDBCConnectionManager jdbcConnectionManager(
     return new JDBCConnectionManager(
         connectionConfig.getUrl(),
         connectionConfig.getMaxNumRetries(),
+        connectionConfig.getVerboseLogging(),
```

Comment:
We could consistently use `showWarnings` instead of a mix of `verbose` and `showWarnings` in different parts of the code.

Reply:
It seems this was over an old version. Resolving.